<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartDialog</class>
 <widget class="QDialog" name="StartDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>225</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Game</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="gameTimeBlock">
     <property name="title">
      <string>Time</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QSpinBox" name="timeHour">
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="suffix">
         <string> h</string>
        </property>
        <property name="maximum">
         <number>11</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="timeMin">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="suffix">
         <string> min</string>
        </property>
        <property name="maximum">
         <number>59</number>
        </property>
        <property name="value">
         <number>15</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="timeSec">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="suffix">
         <string> sec</string>
        </property>
        <property name="prefix">
         <string/>
        </property>
        <property name="maximum">
         <number>59</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="modeBlock">
     <property name="title">
      <string>Mode</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="modeOnePlayer">
        <property name="text">
         <string>1 player</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="modeTwoPlayers">
        <property name="text">
         <string>2 players</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="modeAi">
        <property name="text">
         <string>AI</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="networkBlock">
     <property name="title">
      <string>Network</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="labelNetwork">
        <property name="text">
         <string>IP:port</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="netParams">
        <property name="inputMethodHints">
         <set>Qt::ImhDialableCharactersOnly</set>
        </property>
        <property name="inputMask">
         <string>000.000.000.000:00000</string>
        </property>
        <property name="text">
         <string>127.0.0.0:5000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="optionalBlock">
     <property name="title">
      <string>Optional</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QPushButton" name="jsonButton">
        <property name="text">
         <string>Select JSON</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="historyButton">
        <property name="text">
         <string>Select History</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="statusLabel">
        <property name="font">
         <font>
          <family>Arial</family>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Status: not loaded</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="ok">
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
